packages/node-sdk/paima-engine/scripts/copyAssets.cjs

```diff
@@ -0,0 +1,32 @@
+function copyAssetsForBundle(outDir) {
```

Comment:
`@paima/build-utils` is meant for stuff like this (so that the build tools can be a devDependency while still keeping the Paima node as a non-dev dependency in package.json)